<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="HomeBut">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>740</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1271</width>
     <height>711</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="TopicButton11">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1.1 The characteristics of contemporary processors, input, 
 output and storage devices</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="TopicButton12">
      <property name="text">
       <string>1.2 Software and software development</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="TopicButton13">
      <property name="text">
       <string>1.3 Exchanging data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="TopicButton14">
      <property name="text">
       <string>1.4 Data types, data structures and algorithms</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="TopicButton15">
      <property name="text">
       <string>1.5 Legal, moral, cultural and ethical issues</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="TopicButton21">
      <property name="text">
       <string>
2.1 Elements of computational thinking</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="TopicButton22">
      <property name="text">
       <string>2.2 Problem solving and programming</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="TopicButton23">
      <property name="text">
       <string>2.3 Algorithms</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="AllTopicButton">
      <property name="text">
       <string>All topics</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
